### PR TITLE
Fix token_id values for whisper export

### PR DESCRIPTION
**Pull request by petermcaughan**
### Description
The current ONNX export of Whisper utilizes hard-coded values for token_ids when configuring the BeamSearch node. This PR removes these literals and instead takes these values straight from the WhisperConfig.

### Motivation and Context
Hard-coding these values can cause some parity issues when comparing to default PyTorch behavior - this change to take from WhisperConfig resolves these.

### CODE REVIEW

**Review by tianleiwu — COMMENTED** [review]
I notice that whisper has the following configuration:
https://huggingface.co/openai/whisper-large/blob/27b5bb3f092ae34d8079fdc073b8195e6815dec7/config.json#L46

And
https://huggingface.co/openai/whisper-large/blob/27b5bb3f092ae34d8079fdc073b8195e6815dec7/config.json#L26

How to handle them in beam search?

**Comment by petermcaughan** [comment]
> I notice that whisper has the following configuration: https://huggingface.co/openai/whisper-large/blob/27b5bb3f092ae34d8079fdc073b8195e6815dec7/config.json#L46
> 
> And https://huggingface.co/openai/whisper-large/blob/27b5bb3f092ae34d8079fdc073b8195e6815dec7/config.json#L26
> 
> How to handle them in beam search?



> I notice that whisper has the following configuration: https://huggingface.co/openai/whisper-large/blob/27b5bb3f092ae34d8079fdc073b8195e6815dec7/config.json#L46
> 
> And https://huggingface.co/openai/whisper-large/blob/27b5bb3f092ae34d8079fdc073b8195e6815dec7/config.json#L26
> 
> How to handle them in beam search?

Good catch - I see they apply these values by setting logit values like this [link to HF](https://github.com/huggingface/transformers/blob/edb704b26e79b6be03061ba45f340f0acd1f8ab1/src/transformers/generation/logits_process.py#L874) after obtaining logits. We could replicate this by adding attributes to the BeamSearch op and doing a similar transformation before processing logits - this could make a good PR & would be relevant to all transformer models it seems.

**Review by tianleiwu — APPROVED** [review]

**Review by yufenglee — APPROVED** [review]
